docs/docs/migration_notes.md

```diff
@@ -8,6 +8,12 @@ Aztec is in full-speed development. Literally every version breaks compatibility
 
 ## TBD
 
+### Changes to public calling convention
+
+Contracts that include public functions (that is, marked with `#[public]`), are required to have a function `public_dispatch(selector: Field)` which acts as an entry point. This will be soon the only public function registered/deployed in contracts. The calling convention is updated so that external calls are made to this function.
+
+If you are writing your contracts using Aztec-nr, there is nothing you need to change. The `public_dispatch` function is automatically generated by the `#[aztec]` macro.
```

Comment:
@critesjosh can you create a github issue for devrel to add this to macros docs?

Reply:
yes will do